hardhat.config.js

```diff
@@ -62,21 +62,21 @@ module.exports = {
         disambiguatePaths: false
     },
     abiExporter: {
-        path: './solidity/build/abi',
+        path: './build/abi',
         clear: true
     },
 
     // System Config
     paths: {
-        sources: './solidity/contracts',
-        tests: './solidity/test',
+        sources: './contracts',
+        tests: './test',
         cache: './cache',
-        artifacts: './solidity/build/contracts'
+        artifacts: './build/contracts'
     },
 
     // Test Config
     mocha: {
-        spec: argv.spec || 'solidity/test',
+        spec: argv.spec || 'test',
```

Comment:
IMO, this is a good opportunity to rename folder `test` (Truffle's default) to `tests`.

Reply:
we should just use the standard terminology in hardhat.

Reply:
👍. I'm moving it to the defaults (`test` remains, and `build` is changed to `artifacts`)